README.md

```diff
@@ -44,9 +44,28 @@ This is an engine used by the Frontend site and requires data from the [MAS CMS]
 
   To access the application locally in your browser, navigate to the port on which you started the application, e.g. [http://localhost:5000/en/tools/money-manager/](http://localhost:5000).
 
+
 [Technical docs](https://github.com/moneyadviceservice/technical-docs/tree/master/fin_cap)
 
 ### To Do
 Update link for the styleguide when the site is deployed.
 
 [Styleguide](http://localhost:3000/styleguide)
+
+## Hooks
+
+This project uses git hooks to ensure every commit follows best practices.
```

Comment:
To be clear - in this context is `best practices` the same as `passing tests`?

I don't see anywhere in the `test.sh` file that specifically looks at/validates commits.

Reply:
@scott-ad-riley yes follow some best practices. How can the test.sh looks and validates commits?

Reply:
My comment was just to clear up my own understanding but that's fine - I'd probably not phrase `rubocop` or `rspec` as best practices. You could argue that having passing tests at each commit is, but I'd be hesitant to actually make that opt-out (just my thoughts though).

Reply:
@scott-ad-riley actually I was thinking of re-phrasing that sentence to be `meets MAS standards` since it prevents the commits being pushed if rubocop or the test fails.  🤷‍♀️

Reply:
Yes, I understand your points. And yes, the passing tests would be my arguments, but I think I should had made more clear about this PR. This PR **is not** a imposed thing that everyone **should** be using, but is more like a reference guide. More like this idea: We have things we agreed about code quality and security so then this hook is available to anyone who sees the benefits and wants to make use of them.

Reply:
Yep - this makes sense, i'm mostly commenting to clear up my understanding.

Would you get the same value by having CI run tests on `push` and `pr`?

Reply:
Not necessarily. Yes, would be good to be on `push` and `pr` but I don't know if is good because this could make some people to leave the total responsibility to the CI.

Reply:
And the worry from that is that the PR will be reviewed/merged with failing tests/rubocop?

Reply:
Sorry, again I am not explaining well. Apologies. This project will eventually have the CI with `push` and `PR`. So the PR will not have nothing failing before reviewed/merged. But this hook is a facilitator to anyone who wants to use and catch much before even reaching the CI. Every problem that can we detect as soon as possible on the process is good. And the good thing whoever that is using this hook is that: will avoid the PR review to be convoluted with codestyle comments and more on what supposed to be like code logic, organisation, object oriented design and maintainable.